src/OpenTelemetry.ResourceDetectors.AWS/AWSECSResourceDetector.cs

```diff
@@ -14,6 +14,7 @@
 // limitations under the License.
 // </copyright>
 
+#if !NETFRAMEWORK
```

Comment:
I didn't quite get why is this being excluded for .Net Framework?

Reply:
It is just different way (IMO more clear) to write

```xml
  <ItemGroup Condition="'$(TargetFramework)' == '$(NetFrameworkMinimumSupportedVersion)'">
    <Compile Remove="AWSECSResourceDetector.cs" />
    <Compile Remove="AWSEKSResourceDetector.cs" />
    <Compile Remove="Http\Handler.cs" />
    <Compile Remove="Http\ServerCertificateValidationProvider.cs" />
  </ItemGroup>
```

Based on implementation details, EKS and ECS are looking into unix formatted paths, so there is no reason to have it on .NET Framework. For sure it can be changed, but it should be treated as a separate PR.